Libraries/PeriphDrivers/Include/MAX32655/i2s.h

```diff
@@ -64,20 +64,16 @@ typedef enum { MXC_I2S_MSB_JUSTIFY, MXC_I2S_LSB_JUSTIFY } mxc_i2s_justify_t;
 typedef enum { MXC_I2S_CLKSRC_ERFO, MXC_I2S_CLKSRC_EXT } mxc_i2s_clksrc_t;
 
 /** @brief  I2S transaction word size */
-typedef enum {
-    MXC_I2S_DATASIZE_BYTE,
-    MXC_I2S_DATASIZE_HALFWORD,
-    MXC_I2S_DATASIZE_WORD
-} mxc_i2s_wsize_t;
+typedef enum { MXC_I2S_WSIZE_BYTE, MXC_I2S_WSIZE_HALFWORD, MXC_I2S_WSIZE_WORD } mxc_i2s_wsize_t;
```

Comment:
This change is just for following the MSDK convention - I just thought I2S driver needs a full renewal. I prefer not to apply above suggestion as it adds more complexity, we can keep as it is if you think this would cause compatibility problems. Please let me know.

Reply:
We've implemented these enum "pass-throughs" to avoid compilation errors in some other drivers.  So there is a precedent for doing this.  We must consider projects that are even 3-4 years old.  If we can avoid breaking them, then we must implement it.

I'm in favor of your change to match the enums to the typdef name.  It's annoying when I'm writing code but the enum does not match what I expect.  This also matches the field name from the UG.

I also vote for the additional pass-throughs to preserve backwards compatibility.  We should add comments/doxygen to them, though.

```C
/** @brief I2S transaction word size.  
 * 
 * Set this field to the desired width for data writes and reads from the FIFO.  
 * This should be greater than or equal to the number of bits per word.  */
typedef enum { 
    MXC_I2S_WSIZE_BYTE, ///< Set 8-bit FIFO transactions
    MXC_I2S_WSIZE_HALFWORD, ///< Set 16-bit FIFO transactions
    MXC_I2S_WSIZE_WORD ///< Set 32-bit FIFO transactions

    MXC_I2S_DATASIZE_BYTE = MXC_I2S_WSIZE_BYTE, ///< Legacy name.  Use MXC_I2S_WSIZE_BYTE instead.
    MXC_I2S_DATASIZE_HALFWORD = MXC_I2S_WSIZE_HALFWORD, ///< Legacy name.  Use MXC_I2S_WSIZE_HALFWORD instead.
    MXC_I2S_DATASIZE_WORD = MXC_I2S_WSIZE_WORD, ///< Legacy name.  Use MXC_I2S_WSIZE_WORD instead.
} mxc_i2s_wsize_t;
```